modules/reactor/src/meshgenerators/CoreMeshGenerator.C

```diff
@@ -671,6 +677,8 @@ CoreMeshGenerator::generate()
       auto elem_block_name = default_block_name;
       if (has_block_names)
         elem_block_name += "_" + _pin_block_name_map[pin_type_id][z_id][radial_idx];
+      if (getReactorParam<bool>(RGMB::region_id_as_block_name))
```

Comment:
same

Reply:
I added an else here, if that is what you meant